pkgs/development/compilers/llvm/common/libunwind/default.nix

```diff
@@ -67,6 +67,8 @@ stdenv.mkDerivation (rec {
     ninja python3
   ];
 
+  LDFLAGS = "-unwindlib=none";
+
```

Comment:
this seems out of place -- most / all options for the llvm toolchains are selected via cmake defines. why is this one different, and why is it ok for it to be unconditional. there needs to be a comment providing why this is required.

Reply:
Yeah, I think a comment explaining this would be great. I've been trying to use nixpkgs-review but on every system I try it crashes heh.